jinahub/indexers/searcher/compound/FaissPostgresSearcher/faisspsql.py

```diff
@@ -62,29 +65,29 @@ def __init__(
             self._vec_indexer,
         ) = self._init_executors(dump_path, kwargs, startup_sync_args)
         if startup_sync_args:
-            startup_sync_args['startup'] = True
+            startup_sync_args["startup"] = True
             self.sync(parameters=startup_sync_args)
 
     def _init_executors(self, dump_path, kwargs, startup_sync_args):
         # float32 because that's what faiss expects
         kv_indexer = PostgreSQLStorage(dump_dtype=np.float32, **kwargs)
-        vec_indexer = FaissSearcher(dump_path=dump_path, **kwargs)
+        vec_indexer = FaissSearcher(dump_path=dump_path, prefetch_size=12, **kwargs)
```

Comment:
Why 12? Sounds quite arbitrary.

Reply:
512 was the default. I just needed a smaller number. Idk @numb3r3 ?

Reply:
take some power of 2.